src/modules/dashboard/components/ActionsPage/DetailsWidget/DetailsWidgetRoles/DetailsWidgetRolesItem.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { useIntl } from 'react-intl';
+
+import { ActionUserRoles } from '~types/index';
+import PermissionsLabel from '~core/PermissionsLabel';
+
+import styles from './DetailsWidgetRoles.css';
+
+const displayName =
+  // eslint-disable-next-line max-len
+  'dashboard.ActionsPage.DetailsWidget.DetailsWidgetRoles.DetailsWidgetRolesItem';
```

Comment:
If you add backticks ` you won't need to disable line length, as those are ignored by our eslint rules, as we assume they are used for long strings

```suggestion
  `dashboard.ActionsPage.DetailsWidget.DetailsWidgetRoles.DetailsWidgetRolesItem`;
```